index.js

```diff
@@ -104,8 +104,7 @@ export default class LinterJSCS {
         this.jscs.registerDefaultRules();
 
         const filePath = editor.getPath();
-        const config = configFile.load(false,
-          path.join(path.dirname(filePath), this.configPath));
+        const config = configFile.load(false, this.configPath);
```

Comment:
`atom.project.getDirectories()[0].getPath()` will get the root of the project (as Atom sees it) for the first project, which will work for most cases and will follow the current documentation.

It will only fail when the user has more than one project open in a single Atom window.